From f4f495ee212c46611303dec9cd18996830d7f721 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Wed, 20 Nov 2024 10:10:09 +0100
Subject: [PATCH] node: cast proxy to pw_node* when calling pw_node functions

This currently works fine because the functions accept void* but will
fail when they accept struct pw_node* in the future.
---
 lib/wp/node.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/wp/node.c b/lib/wp/node.c
index 07fa0bbe..7b77b944 100644
--- a/lib/wp/node.c
+++ b/lib/wp/node.c
@@ -631,7 +631,7 @@ wp_node_send_command (WpNode * self, const gchar * command)
 
   struct spa_command cmd =
       SPA_NODE_COMMAND_INIT(wp_spa_id_value_number (command_value));
-  pw_node_send_command (wp_proxy_get_pw_proxy (WP_PROXY (self)), &cmd);
+  pw_node_send_command ((struct pw_node*)wp_proxy_get_pw_proxy (WP_PROXY (self)), &cmd);
 }
 
 /*! \defgroup wpimplnode WpImplNode */
-- 
2.51.0

